litejpeg/core/huffman/ac_cr_rom.csv

```diff
@@ -0,0 +1,253 @@
+0b1010,4
```

Comment:
It would be good if this file had some comments at the top describing what this file is used for, why it exists and what the values mean.

To read such a file, you'll need to do something like described here -> https://stackoverflow.com/questions/14158868/python-skip-comment-lines-marked-with-in-csv-dictreader or https://wuhrr.wordpress.com/2016/02/16/process-csv-with-comments/

Reply:
Done